arm/examples/aci/create.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+// Copyright 2017 Microsoft Corporation
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+import (
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/arm/containerinstance"
+	"github.com/Azure/azure-sdk-for-go/arm/examples"
+	"github.com/Azure/go-autorest/autorest"
+)
+
+var (
+	resourceGroupName     = "myresourcegroup"
```

Comment:
I've been thinking of going and refurbishing a bunch of our samples. One of the improvements I was planning on making was to create a resource group with a unique name which gets deleted at the end of the sample by default. I hacked up a little library for creating such names that can be found here: https://github.com/marstr/randname/blob/master/adjnoun.go

What are your thoughts on that approach? Think it would simplify the way folks needed to interact with this sample?